product_docs/docs/migration_toolkit/55/08_mtk_command_options.mdx

```diff
@@ -582,7 +582,7 @@ ALTER TABLE DEPT ADD CONSTRAINT DEPT_DNAME_UQ UNIQUE (DNAME);
 
 When you then migrate this table and create it in EDB Postgres Advanced Server, all unquoted object names are converted to lowercase letters, so the table appears in EDB Postgres Advanced Server as follows:
 
-```text
+```sql
```

Comment:
I don't think we want this one to be sql. It is probably output, but my recommendation is to use text.